locales/en/plugin__kuadrant-console-plugin.json

```diff
@@ -78,19 +77,15 @@
   "Policy Topology": "Policy Topology",
   "Port": "Port",
   "Protocol": "Protocol",
+  "Provider Ref": "Provider Ref",
   "RateLimit": "RateLimit",
-  "RateLimitPolicy enables rate limiting for service workloads in a Gateway API network": "RateLimitPolicy enables rate limiting for service workloads in a Gateway API network",
+  "RateLimitPolicy": "RateLimitPolicy",
   "Read about the latest information and key features in the Kuadrant highlights.": "Read about the latest information and key features in the Kuadrant highlights.",
   "Release Notes": "Release Notes",
-  "Remove Custom Selector": "Remove Custom Selector",
-  "Remove Match Expression": "Remove Match Expression",
-  "Remove Match Label": "Remove Match Label",
-  "Remove Value": "Remove Value",
-  "Routing Strategy": "Routing Strategy",
-  "Routing Strategy to use": "Routing Strategy to use",
   "Save": "Save",
   "Select a gateway": "Select a gateway",
-  "Select a Namespace": "Select a Namespace",
+  "Select a protocol": "Select a protocol",
+  "Select a Protocol": "Select a Protocol",
```

Comment:
2 feels like a bit overkill

Reply:
Got to make absolutely sure

(this was generated by the `yarn i18n` task, looks like an oddity. I'll replace one of them.)

Reply:
fixed